test/Chirp.Infrastructure.Tests/UnitTest.cs

```diff
@@ -31,6 +31,87 @@ public class UnitTest
         act.Should().Throw<Exception>().WithMessage("Message is too long");
     }
 
+    [Fact]
+    public void TestRestrictedCreationOfCheepAt160Char(){
+        //Arrange
+        //Runs in memory
+        using var connection = new SqliteConnection("Filename=:memory:");
+        var builder = new DbContextOptionsBuilder<ChirpDBContext>();
+        builder.UseSqlite(connection);
+        var context = new ChirpDBContext(builder.Options);
+        context.Database.EnsureCreated();
+        var repository = new CheepRepository(context);
+        connection.Open();
+
+        //Act
+        CheepDTO cheep = new()
+        {
+            AuthorId = 3,
+            Author = "fjkd",
+            Timestamp = DateTime.Now.ToString(),
+            Message = " This string should be at exactly 160 characters, so that we know its possible. There should not be a character more or less, so we'll have a very precise test."
+        };
+
+        Action act = () => repository.AddCheep(cheep.AuthorId, cheep.Message);
+
+        //Assert
+        act.Should().NotThrow<Exception>();
+    }
+
+     [Fact]
+    public void TestRestrictedCreationOfCheepUnder160Char(){
+        //Arrange
+        //Runs in memory
+        using var connection = new SqliteConnection("Filename=:memory:");
+        var builder = new DbContextOptionsBuilder<ChirpDBContext>();
+        builder.UseSqlite(connection);
+        var context = new ChirpDBContext(builder.Options);
+        context.Database.EnsureCreated();
+        var repository = new CheepRepository(context);
+        connection.Open();
+
+        //Act
+        CheepDTO cheep = new()
+        {
+            AuthorId = 3,
+            Author = "fjkd",
+            Timestamp = DateTime.Now.ToString(),
+            Message = "This string is very much under 160 characters"
+        };
+
+        Action act = () => repository.AddCheep(cheep.AuthorId, cheep.Message);
+
+        //Assert
+        act.Should().NotThrow<Exception>();
+    }
+
+     [Fact]
+    public void TestRestrictedCreationOfCheepOf0Char(){
+        //Arrange
+        //Runs in memory
+        using var connection = new SqliteConnection("Filename=:memory:");
+        var builder = new DbContextOptionsBuilder<ChirpDBContext>();
+        builder.UseSqlite(connection);
+        var context = new ChirpDBContext(builder.Options);
+        context.Database.EnsureCreated();
+        var repository = new CheepRepository(context);
+        connection.Open();
+
+        //Act
+        CheepDTO cheep = new()
+        {
+            AuthorId = 3,
+            Author = "fjkd",
+            Timestamp = DateTime.Now.ToString(),
+            Message = ""
+        };
+
+        Action act = () => repository.AddCheep(cheep.AuthorId, cheep.Message);
+
+        //Assert
+        act.Should().Throw<Exception>().WithMessage("Message is empty");
```

Comment:
What kind of assert is this ?

Reply:
I have found the documentation here:
https://fluentassertions.com/introduction